ethereum/core/src/main/java/org/hyperledger/besu/ethereum/core/PrivacyParameters.java

```diff
@@ -201,6 +204,21 @@ public Builder setMultiTenancyEnabled(final boolean multiTenancyEnabled) {
       return this;
     }
 
+    public Builder setPrivacyKeyStoreFile(final Path privacyKeyStoreFile) {
+      this.privacyKeyStoreFile = privacyKeyStoreFile;
+      return this;
+    }
+
+    public Builder setPrivacyKeyStorePasswordFile(final Path privacyKeyStorePasswordFile) {
+      this.privacyKeyStorePasswordFile = privacyKeyStorePasswordFile;
+      return this;
+    }
+
+    public Builder setPrivacyEnclaveWhitelistFile(final Path privacyEnclaveWhitelistFile) {
```

Comment:
rename to setPrivacyTlsKnownEnclaveFile to match the command line name change

Reply:
done